src/openrct2/management/Research.h

```diff
@@ -97,7 +97,7 @@ struct ResearchItem
         else
         {
             retItem.entryIndex = OpenRCT2EntryIndexToRCTEntryIndex(entryIndex);
-            retItem.baseRideType = OpenRCT2RideTypeToRCT2RideType(baseRideType);
+            retItem.baseRideType = RCT2::OpenRCT2RideTypeToRCT2RideType(baseRideType);
```

Comment:
To be deleted. Function unused https://github.com/OpenRCT2/OpenRCT2/issues/16013